templates/taxi/manufacturer_delete_confirm.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Are you sure you want to delete "{{ object }}"</h1>
+  <ul><li>this action will remove all associated cars</li></ul>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input class="btn btn-danger" type="submit" value="Delete" />

```

Comment:
According to checklist, not bad will be add Cancel button apart from Delete one.